core/equations.gms

```diff
@@ -1127,4 +1127,14 @@ q_shbiofe_lo(t,regi,entyFe,sector,emiMkt)$(pm_secBioShare(t,regi,entyFe,sector)
   sum((entySeBio,te)$se2fe(entySeBio,entyFe,te), vm_demFeSector_afterTax(t,regi,entySeBio,entyFe,sector,emiMkt))
 ;
 
+***---------------------------------------------------------------------------
+*' Limit solids fossil to be lower or equal to previous year values  
+***---------------------------------------------------------------------------
+$ifthen.limitSolidsFossilRegi not %cm_limitSolidsFossilRegi% == "off"
+q_fossilSolidsLimitReg(ttot,regi,entySe,entyFe,sector,emiMkt)$(limitSolidsFossilRegi(regi) and (ttot.val ge max(2020, cm_startyear)) AND sefe(entySe,entyFe) AND sector2emiMkt(sector,emiMkt) AND (sameas(sector,"indst") OR sameas(sector,"build")) AND sameas(entySe,"sesofos"))..
```

Comment:
this is on for ttot = 2020, so that in 2020 fossil solids use already has to go down. 
I guess we are positive that the average of 2018-2022 is already lower than the average 2013-2017 in each EU region and US historically?

Reply:
This could be an small issue for FRA, and it would be a real issue if anybody enforces this to India.

![image](https://github.com/user-attachments/assets/dccc1d71-bc1d-4cab-a7a4-55d20094dbca)

I will check if we can control the historical data already by some of our other equations instead.

Reply:
If you are ok with it, I would merge as is for now as I only enable this for EU27 in my paper runs, and I will check on enforcing historical values after I submit the paper revision.

Reply:
sure, merge for now. 
just wondering: what would speak against setting the starting point to 2025? I don't think the runs ever produced a solids spike in 2020, only in 2025?